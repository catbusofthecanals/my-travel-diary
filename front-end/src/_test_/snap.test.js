import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Register from "../components/Registration";
//import BrowserRouter to successfully render <Link> in heading

// snapshot test of help component
test("renders correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Register />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
