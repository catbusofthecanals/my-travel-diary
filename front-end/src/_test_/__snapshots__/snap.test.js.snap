// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <div
    className="header row"
  >
    <div
      className="text-center"
    >
      <img
        alt="logo icon"
        className="header-img"
        src="logo.png"
        width="150px"
      />
      <h2>
        My Travel Diary
      </h2>
    </div>
  </div>
  <div
    className="space-above row"
  >
    <div
      className="text-center"
    >
      <form
        onSubmit={[Function]}
      >
        <p
          className="has-text-centered"
        />
        <div
          className="field mt-5"
        >
          <label
            className="label"
          >
            Name
          </label>
          <div
            className="controls"
          >
            <input
              className="input"
              onChange={[Function]}
              placeholder="Username"
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="field mt-5"
        >
          <label
            className="label"
          >
            Email
          </label>
          <div
            className="controls"
          >
            <input
              className="input"
              onChange={[Function]}
              placeholder="Email"
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="field mt-5"
        >
          <label
            className="label"
          >
            Password
          </label>
          <div
            className="controls"
          >
            <input
              className="input"
              onChange={[Function]}
              placeholder="******"
              type="password"
              value=""
            />
          </div>
        </div>
        <div
          className="field mt-5"
        >
          <label
            className="label"
          >
            Confirm Password
          </label>
          <div
            className="controls"
          >
            <input
              className="input"
              onChange={[Function]}
              placeholder="******"
              type="password"
              value=""
            />
          </div>
        </div>
        <div
          className="field mt-5"
        >
          <button
            className="button is-success is-fullwidth"
          >
            Register
          </button>
        </div>
      </form>
    </div>
  </div>
</section>
`;
